src/smclarify/bias/metrics/basic_stats.py

```diff
@@ -39,3 +35,191 @@ def proportion(sensitive_facet_index: pd.Series) -> float:
+    Proportion of inputs with negative observed label correctly assigned the negative predicted label.
+
+    :param: FP Counts of labels which were incorrectly predicted positive
+    :param: TN Counts of labels which were correctly predicted negative
+    :return: Proportion of inputs with negative observed label correctly assigned the negative predicted label.
+    """
+    return divide(TN, TN + FP)
+
+
+def precision(TP: int, FP: int) -> float:
+    r"""
+    Proportion of inputs with positive predicted label that actually have a positive observed label.
+
+    :param: TP Counts of labels which were correctly predicted positive
+    :param: FP Counts of labels which were incorrectly predicted positive
+    :return: Proportion of inputs with positive predicted label that actually have a positive observed label.
+    """
+    return divide(TP, FP + TP)
+
+
+def rejection_rate(TN: int, FN: int) -> float:
+    r"""
+    Proportion of inputs with negative predicted label that actually have a negative observed label.
+
+    :param: TN Counts of labels which were correctly predicted negative
+    :param: FN Counts of labels which were incorrectly predicted negative
+    :return: Proportion of inputs with negative predicted label that actually have a negative observed label.
+    """
+    return divide(TN, TN + FN)
+
+
+def conditional_acceptance(TP: int, FP: int, FN: int) -> float:
+    r"""
+    Ratio between the positive observed labels and positive predicted labels.
+
+    :param: TP Counts of labels which were correctly predicted positive
+    :param: FP Counts of labels which were incorrectly predicted positive
+    :param: FN Counts of labels which were incorrectly predicted negative
+    :return: Ratio between the positive observed labels and positive predicted labels.
+    """
+    return divide(FN + TP, FP + TP)
+
+
+def conditional_rejection(FP: int, TN: int, FN: int) -> float:
+    r"""
+    Ratio between the negative observed labels and negative predicted labels.
+
+    :param: FP Counts of labels which were incorrectly predicted positive
+    :param: TN Counts of labels which were correctly predicted negative
+    :param: FN Counts of labels which were incorrectly predicted negative
+    :return: Ratio between the negative observed labels and negative predicted labels.
+    """
+    return divide(TN + FP, TN + FN)
+
+
+def f1_score(TP: int, FP: int, FN: int) -> float:
+    r"""
+    Harmonic mean of precision and recall.
+
+    :param: TP Counts of labels which were correctly predicted positive
+    :param: FP Counts of labels which were incorrectly predicted positive
+    :param: FN Counts of labels which were incorrectly predicted negative
+    :return: Harmonic mean of precision and recall.
+    """
+    precision_score = precision(TP, FP)
+    recall_score = recall(TP, FN)
+    return 2 * divide(precision_score * recall_score, precision_score + recall_score)
+
+
+# Model Performance Metrics
+def multicategory_confusion_matrix(
```

Comment:
would be good to add an example in the docstring for reference, as I had to look at the test to fully understand it.